package.json

```diff
@@ -10,18 +10,23 @@
   "scripts": {
     "build": "node_modules/.bin/webpack",
     "start": "node_modules/.bin/webpack-dev-server --host 0.0.0.0 --content-base public/ --hot",
-    "release": "node_modules/.bin/webpack -p"
+    "release": "node_modules/.bin/webpack",
+    "lint": "node_modules/.bin/eslint index.js"
   },
   "author": "Clappr team",
   "license": "BSD-3-Clause",
-  "dependencies": {
-    "shaka-player": "v2.0.9",
-    "clappr": "0.2.66"
-  },
   "devDependencies": {
-    "webpack": "^1.12.1",
-    "webpack-dev-server": "^1.10.1",
+    "shaka-player": "2.0.6",
```

Comment:
`v2.0.9` doesn't exist in NPM, it doesn't seem to be a valid release actually? 

`2.0.8` is the latest release on the `2.0.x` minor.